data/json/items/containers.json

```diff
@@ -1140,6 +1140,26 @@
     "seals": true,
     "properties": [ [ "burst_when_filled", "75" ] ]
   },
+  {
+    "id": "balloon",
+    "type": "CONTAINER",
+    "category": "other",
+    "looks_like": "condom",
+    "name": "balloon",
+    "description": "A child's balloon.  This could be used as a makeshift water container.",
+    "weight": "5 g",
+    "volume": 0,
+    "price": 0,
+    "to_hit": -5,
+    "rigid": false,
+    "material": "plastic",
+    "symbol": ")",
+    "color": "white",
+    "contains": "3750 ml",
+    "watertight": true,
+    "seals": true,
+    "properties": [ [ "burst_when_filled", "75" ] ]
```

Comment:
A lot of these entries could be removed if you add `"copy-from": "condom"`.
Or perhaps go the other direction and have condom inherit from balloon.